packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -1024,6 +1079,10 @@ export class AppManagementClient implements DeveloperPlatformClient {
     )
   }
 
+  private createUnauthorizedHandler(): UnauthorizedHandler {
+    return createUnauthorizedHandler(this)
+  }
```

Comment:
this is not being used, all functions call the external function not the private one. `createUnauthorizedHandler(this)`